lib/lighthouse/veterans_health/client.rb

```diff
@@ -30,6 +30,22 @@ def initialize(icn)
         raise ArgumentError, 'no ICN passed in for LH API request.' if icn.blank?
       end
 
+      def list_diagnostic_reports(params = {})
```

Comment:
What's the purpose of the leading underscores? My first thought was "they're not used" but it seems that's what the upstream expects.

Reply:
Ya I have no idea. That's why I changed them in the query params as it doesn't make sense to me that upstream wants the underscore